src/EditorFeatures/CSharpTest/Diagnostics/UpgradeProject/UpgradeProjectTests.cs

```diff
@@ -51,7 +51,7 @@ void A()
         var x = [|(1, 2)|];
     }
 }",
-                LanguageVersion.Default,
+                LanguageVersion.Default.MapSpecifiedToEffectiveVersion(),
```

Comment:
I'd remove this test entirely, since duplicate of one below (`UpgradeProjectFromCSharp6ToCSharp7`)